crates/nu-parser/src/parse_keywords.rs

```diff
@@ -3544,6 +3544,17 @@ pub fn parse_source(working_set: &mut StateWorkingSet, lite_command: &LiteComman
                             ),
                         );
 
+                        // store the file path as a string to be gathered later
+                        call_with_block.set_parser_info(
+                            "block_id_name".to_string(),
+                            Expression::new(
```

Comment:
Just a general comment, but I really need to get around to making parser info *not* be expressions (it's an issue for IR...)

Reply:
Ya, I wasn't sure if there was some other way to do this, so I just copied what was there before.